<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>StockTrack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Вкладка 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Вкладка 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Экспортировать</string>
     </property>
     <addaction name="exportExcel"/>
    </widget>
    <addaction name="menu_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>О программе</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="Menu">
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="contractors"/>
   <addaction name="units_of_measurmant"/>
   <addaction name="category"/>
   <addaction name="status_goods"/>
   <addaction name="goods"/>
   <addaction name="status_realization"/>
   <addaction name="orders"/>
   <addaction name="shipment"/>
   <addaction name="realization"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="add_record"/>
   <addaction name="delete_record"/>
   <addaction name="add_tab"/>
   <addaction name="search"/>
   <addaction name="report_action"/>
  </widget>
  <action name="contractors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Поставщики</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Контрагенты&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="units_of_measurmant">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Единицы измерения</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="category">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Категории</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Категории&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="status_goods">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Статус товара</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="goods">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Товары</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="status_realization">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Статус реализации</string>
   </property>
   <property name="toolTip">
    <string>Статус реализации</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="orders">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Заказы</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="shipment">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Поставки</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="Role">
   <property name="text">
    <string>Роли</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="Rights">
   <property name="text">
    <string>Права</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="Inf_programm">
   <property name="text">
    <string>Информация о программе</string>
   </property>
  </action>
  <action name="realization">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Реализации</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="add_record">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Action_table_img/add.png</normaloff>:/image/Action_table_img/add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить запись</string>
   </property>
  </action>
  <action name="delete_record">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Action_table_img/delete.png</normaloff>:/image/Action_table_img/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить запись</string>
   </property>
  </action>
  <action name="add_tab">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Action_table_img/add_tab.png</normaloff>:/image/Action_table_img/add_tab.png</iconset>
   </property>
   <property name="text">
    <string>Добавить вкладку</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="remove_tab">
   <property name="text">
    <string>Удалить вкладку</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="exportExcel">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Excel</string>
   </property>
   <property name="toolTip">
    <string>Excel</string>
   </property>
  </action>
  <action name="search">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Action_table_img/search.png</normaloff>:/image/Action_table_img/search.png</iconset>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="report_action">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Action_table_img/report.png</normaloff>:/image/Action_table_img/report.png</iconset>
   </property>
   <property name="text">
    <string>Отчет</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
